systemtests/utils/vagrant.go

```diff
@@ -75,11 +76,52 @@ func (v *Vagrant) Setup(env string, numNodes int) error {
 		return err
 	}
 
+	// some more work to figure the ssh port and private key details
```

Comment:
we could throw away all of this if we used the vagrant insecure keys, and then just bundled that private key in our repository or on the vagrant image.